src/utils/daos_hdlr.c

```diff
@@ -168,6 +168,34 @@ pool_decode_props(struct cmd_args_s *ap, daos_prop_t *props)
 		}
 	}
 
+	entry = daos_prop_entry_get(props, DAOS_PROP_PO_EC_PDA);
```

Comment:
Not positive about this, but I don't think this code is in use anymore. This is the handler for the old version of the daos tool.

Reply:
Yup, i don't see any usage of functions in daos_hdlr.c, do you think it ok i remove this file directly?

Reply:
I'd be inclined to say yes, if we don't reference it anymore, we should remove it.

@mjmac Is there a reason we were holding onto this file after the daos tool conversion to Go?

Reply:
If nothing is using it anymore, then I think it's fine to remove it. @ashleypittman, @mchaarawi: Any problems with that?

Reply:
nope. the old daos tool was removed, and we can remove all hdlr code that is not used by the daos go tool.

Reply:
I had a PR to remove it at one point but was told that parts of it were actually still in use.  I'll try and find the PR to see what the user was.

Reply:
I see this some usage from daos_autotest.c eg:
[root@server daos]# git grep daos_hdlr.h
src/control/cmd/daos/README.md:In src/utils/daos_hdlr.h:
src/control/cmd/daos/util.h:#include "daos_hdlr.h"
src/utils/daos_autotest.c:#include "daos_hdlr.h"
src/utils/daos_dfs_hdlr.c:#include "daos_hdlr.h"
src/utils/daos_hdlr.c:#include "daos_hdlr.h"
src/utils/daos_obj_ctl.c:#include "daos_hdlr.h"
src/utils/daos_obj_ctl.h:#include "daos_hdlr.h"